src/monobase/pget.py

```diff
@@ -17,6 +17,7 @@
 FUSE_MOUNT = os.environ.get('FUSE_MOUNT', '/srv/r8/fuse-rpc')
 PROC_FILE = os.path.join(FUSE_MOUNT, 'proc', 'pget')
 PGET_CACHED_PREFIXES = os.environ.get('PGET_CACHE_URI_PREFIX', '')
+KNOWN_WEIGHTS_DIR = os.environ.get('KNOWN_WEIGHTS_DIR', '')
```

Comment:
Maybe prefix this with `PGET_` just to be clearer as this will be exposed to model pods.